README.md

```diff
@@ -1,3 +1,34 @@
+
+It is impossible to legislate perfection so this CoC is not designed to outline every bannable offense or lay out, in specifity, an exhaustive list of what is and isn't allowed. The goal is much more modest: to make sure that members understand the spirit they should uphold and to avoid any surprises in the unfortunate circumstance that someone gets removed from the channel for violating that spirit. 
+
+To date, the vast majority of participants have fostered an incredible atmosphere of collaboration, with members offering advice, feedback, etc and we expect that will continue. There have been rare occurences of flagrant or derogatory language being directed at other members. There have been threats of violence. This behavior has no place on the team. 
+
+Clear-cut and eggregious violations may result in immediate removal from the team without warning. If there's an isolated incident followed by a quick resolution or apology to the point where it's clear to any sane, objective third-party that the offending members' continued presence on the team presents no substantial impediment to continued productive discussion focused on FP, we hope a warning will suffice and we can all move on. 
+
+What _precisely_ qualifies as "resolution" or "apology"? We do not attempt to precisely define those terms because this CoC is not intended to be a sprawling regulatory framework which would require lawyers to interpret prior to enforcement. Instead, we've placed significant trust in the few admins and other long-standing key members to assess situations on a case-by-case basis. 
+
+In a word, if you are removed from the team, it means you blatantly violated very basic, very reasonable expectations of good behavior and you're kidding yourself if you think otherwise. 
+
+Some _examples_ (but NOT an exhaustive list of bad behavior):
+
+- racist language directed at someone
+- sexist language directed at someone 
+- homophobic language directed at someone 
+- any other derogation of a person on the basis of their identity of any kind 
+- "fuck you" directed at someone (not said in jest)
+- "I'm going to kill you" (not said in jest) or any other threats of violence
+- monopolizing channels with personal conflicts to the point where it dominates the conversations and affects the ability of members to focus on the channel topic
+
+All but the last of these apply to communications via Direct Messages as well, not just channels. 
+
+None of the admins want to be dealing with enforcement of this CoC or policing activity in the channels. It's annoying and the majority of the members are here to engage in productive conversation. Isolated incidinces rarely end up being the issue. It's when we hear several reports about a member engaging in a pattern of behavior which detracts from the welcoming environment that has been in place since inception. 
```

Comment:
spelling: incidences
non-spelling: last sentence refers to something not mentioned which I assume is being banned but it might read better with "… that a member is removed"

Reply:
Agree that tweaking that last sentence (at minimum, adding that) would make this much more clear.